.eslintrc.json

```diff
@@ -26,7 +26,7 @@
     "no-implicit-globals": 0,
     "no-invalid-this": 0,
     "no-alert": 0,
-    "new-cap": ["error", {
+    "new-cap": ["warn", {
```

Comment:
@HenriRabalais is this gonna give us trouble if it goes back to "error" when we upgrade ?

Reply:
It could, yes. I believe this was done because of some sort of odd react naming convention.